app/javascript/components/Navigation/TradeLogger/index.jsx

```diff
@@ -53,7 +53,7 @@ const TradeLogger = ({ unlockedUsers, currentUserId }) => {
       {xhrResponse && (
         <StatusMessage
           status={xhrResponse.status}
-          invalidTradeTargets={xhrRequest.invalid_trade_targets}
+          invalidTradeTargets={xhrResponse.invalid_trade_targets}
```

Comment:
Just a dumb typo which was causing the trade drawer to close without any notification if there was an error processing the trade.